scss/widgets/generic/gantt/_sizes.scss

```diff
@@ -17,7 +17,6 @@ $gantt-task-wrapper-padding-top: 0;
 @if $size == "default" {
   $gantt-task-title-padding-left: 7px;
   $gantt-task-height: 21px;
-  $gantt-parent-task-height: 19px;
```

Comment:
Why should we remove this?

Reply:
Unused variable

Reply:
Ok. So, should we remove this line as well? https://github.com/DevExpress/DevExtreme/pull/13932/files/92cb5f0fd6068f1a98025bfea05708e486bf67ae#diff-da9a39ef2955420a3c08024f14faa5deL7

Reply:
removed